components/admin_console/admin_definition.jsx

```diff
@@ -844,64 +843,226 @@ export default {
+                        help_text: t('admin.environment.smtp.skipServerCertificateVerification.description'),
+                        help_text_default: 'When true, Mattermost will not verify the email server certificate.',
+                    },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.EnableSecurityFixAlert',
+                        label: t('admin.environment.smtp.enableSecurityFixAlert.title'),
+                        label_default: 'Enable Security Alerts:',
+                        help_text: t('admin.environment.smtp.enableSecurityFixAlert.description'),
+                        help_text_default: 'When true, System Administrators are notified by email if a relevant security fix alert has been announced in the last 12 hours. Requires email to be enabled.',
+                    },
+                ],
+            },
+        },
         email_notifications: {
-            url: 'environment/notifications_email',
+            url: 'environment/email_notifications',
             title: t('admin.sidebar.emailNotifications'),
             title_default: 'Email Notifications',
             isHidden: it.configIsTrue('ExperimentalSettings', 'RestrictSystemAdmin'),
-            searchableStrings: [
-                'admin.notifications.email',
-                'admin.email.notification.contents.title',
-                'admin.email.notificationsTitle',
-                'admin.email.notificationsDescription',
-                'admin.email.enablePreviewModeBannerTitle',
-                'admin.email.enablePreviewModeBannerDescription',
-                'admin.email.enableEmailBatchingTitle',
-                'admin.email.enableEmailBatchingDesc',
-                'admin.email.notificationDisplayTitle',
-                'admin.email.notificationDisplayDescription',
-                'admin.email.notificationEmailTitle',
-                'admin.email.notificationEmailDescription',
-                'admin.email.replyToAddressTitle',
-                'admin.email.replyToAddressDescription',
-                'admin.email.notificationOrganization',
-                'admin.email.notificationOrganizationDescription',
-                'admin.email.smtpServerTitle',
-                'admin.email.smtpServerDescription',
-                'admin.email.smtpPortTitle',
-                'admin.email.smtpPortDescription',
-                'admin.email.enableSMTPAuthTitle',
-                'admin.email.enableSMTPAuthDesc',
-                'admin.email.smtpUsernameTitle',
-                'admin.email.smtpUsernameDescription',
-                'admin.email.smtpPasswordTitle',
-                'admin.email.smtpPasswordDescription',
-                'admin.email.skipServerCertificateVerification.title',
-                'admin.email.skipServerCertificateVerification.description',
-                'admin.service.securityTitle',
-                'admin.service.securityDesc',
```

Comment:
We can removed these translation IDs from `en.json` if not used elsewhere.

Reply:
Ah, good point. Some were probably renamed, but the old names should be removed if they haven't been already.